CONTRIBUTING.md

```diff
@@ -407,6 +411,18 @@ Breakpoints will work as long as you are debugging code in the same package.
 
 > Source map support is partial at this point but actively being worked on.
 
+### Debugging the CI server through the ssh
+
+#### Getting into the SSH connection
+  Upload your public key onto github if not done so already. A public key is necessary to join the ssh connection to use upterm. A comprehensive guide is linked here (insert link here).
+
+  Edit the `ci.yml` file by following the steps here (insert link) to create a upterm session within `.github/workflows` by adding a new build step. 
```

Comment:
```yaml
    - name: Setup upterm session
      uses: lhotari/action-upterm@v1
      with: 
        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Make sure to mention that the SSH upterm session has to come after the checkout step:
- uses: actions/checkout@v2.3.4

Reply:
got it! tytyvm :)